doc/source/examples/multiphysics/water-injection-in-a-closed-cell/water-injection-in-a-closed-cell.rst

```diff
@@ -0,0 +1,308 @@
+================================
+Water Injection in a Closed Cell
+================================
+
+This example simulates the compression of air in a closed cell by injection of water.
+The problem is inspired by the test case of Caltagirone *et al.* `[1] <https://doi.org/10.1016/j.compfluid.2011.06.011>`_
+
+
+--------
+Features
+--------
+
+- Solver: ``gls_navier_stokes`` (with Q1-Q1)
+- Volume of fluid (VOF)
+- Isothermal compressible fluid
+- Usage of a python script for post-processing data
+
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+- Parameter file: ``examples/multiphysics/water-injection-in-a-closed-cell/water-injection-in-a-closed-cell.prm``
+- Python script for postprocessing: ``examples/multiphysics/air-bubble-compression/air-bubble-compression-postprocessing.py``
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+A square-shaped cell with sides of length :math:`L=0.1` is initially filled with air.
+At :math:`t=0 \, \text{s}`, water starts entering the cell through the bottom side at a constant rate of :math:`\mathbf{v}=[0, 0.1]` causing the air in the cell to compress.
+The initial configuration of this example is illustrated below.
+
++-------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/water-injection-initial-configuration.svg                                                     |
+|     :align: center                                                                                                |
+|     :width: 700                                                                                                   |
+|     :name: Water injection initial configuration                                                                  |
+|                                                                                                                   |
+|     Initial configuration                                                                                         |
+|                                                                                                                   |
++-------------------------------------------------------------------------------------------------------------------+
+
+.. note::
+  In this example, gravity force is not considered.
+
+--------------
+Parameter File
+--------------
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~
+
+The time integration is handled by a 1st-order backward differentiation scheme (``bdf1``) with a variable time step is used for time integration.
```

Comment:
Is there a reason for the bdf1?

Reply:
When I initially tested I didn't notice much of a difference in the results, but now I see that the interface is defined better with bdf2, I'll change it back :)